content/docs/2.4/scalers/solace.md

```diff
@@ -0,0 +1,104 @@
++++
+title = "Solace PubSub+"
+availability = "2.3+"
+maintainer = "Community"
+description = "Scale applications based on Solace PubSub+ Broker Queues"
+layout = "scaler"
+go_file = "solace_scaler"
++++
+
+### Trigger Specification
+This specification describes the `solace-queue` trigger that scales based on a Solace PubSub+ queues.
+
+```yaml
+triggers:
+- type: solace-queue
+  metadata:
+    solaceSempBaseURL:    http://solace_broker_semp:8080
+    msgVpn:               message-vpn
+    queueName:            queue_name
+    msgCountTarget:       '100'
+    msgSpoolUsageTarget:  '10000'
+    username:             semp-user
+    password:             semp-pwd
+    usernameEnv:          ENV_VAR_USER
+    passwordEnv:          ENV_VAR_PWD
+```
+
+**Parameter list:**
+- `solaceSempBaseURL` - Solace SEMP Endpoint in format: `<protocol>://<host-or-service>:<port>`
+- `msgVpn` - Message VPN hosted on the Solace broker
```

Comment:
We typically don't abbreviate things
```suggestion
- `messageVpn` - Message VPN hosted on the Solace broker
```

Reply:
Done - Corresponding code change pending